src/test/js/dynamic_user_registration_test.js

```diff
@@ -0,0 +1,60 @@
+var TestData = require('./config/test_data');
+
+Feature('I am able to register user dynamically');
+
+let adminEmail;
+let randomUserFirstName;
+let randomUserLastName;
+let userEmail;
+let accessToken;
+
+const serviceName = 'TEST_SERVICE_' + Date.now();
+const testMailSuffix = '@mailtest.gov.uk';
```

Comment:
Just a comment: are these values consistent with other test parameters we're using? Or do they need to be differentiated in some way?